### PR TITLE
remove size assertions from hyper/hypoplastic

### LINKED ISSUE #489: 'hyperplastic' should be defined by cell number rather than size
Discussed in phenotype ontology editors' call 02/24/22

'hyperplastic' - PATO:0000644 is currently defined to be an increase in size, but this is not always the case (if the cells are smaller). It should instead be defined as an increase in the number of cells.

A similar change should be made to 'hypoplastic' - PATO:0000645

### PR DESCRIPTION
**Pull request by Clare72**
fixes #489

### CODE REVIEW

File: src/ontology/pato-edit.obo [hunk 1]
```diff
@@ -4118,25 +4118,25 @@ is_a: PATO:0001598 ! protruding
 id: PATO:0000644
 name: hyperplastic
 alt_id: PATO:0000943
-def: "A increased size quality inhering in an organ or tissue by virtue of the bearer's exhibiting increased number of cells." [Wikipedia:http\://en.wikipedia.org/wiki/Hyperplastic]
+def: "A physical quality inhering in an organ or tissue by virtue of the bearer's exhibiting an increased number of cells, typically associated with an increase in size." [Wikipedia:Hyperplasia]
```
Comment (rays22) [inline src/ontology/pato-edit.obo:4121]:
```suggestion
def: "A quality inhering in an organ or tissue by virtue of the bearer's exhibiting an increased number of cells, typically associated with an increase in size." [Wikipedia:Hyperplasia]
```
Is *'physical'* necessary in the definiton? 'hyperplastic' is not a child of [PATO:0001241 physical object quality](https://www.ebi.ac.uk/ols/ontologies/pato/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FPATO_0001241&lang=en&viewMode=PreferredRoots&siblings=false).

**Review by rays22 — CHANGES_REQUESTED** [review]
I would drop the adjective *physical* in the text definition.
The rest looks fine to me.

File: src/ontology/pato-edit.obo (hunk 1 above)
Comment (Clare72) [reply src/ontology/pato-edit.obo:4121]: yes it is...
but I guess we don't need to specify it in the definition if people object(?)

**Review by sbello — APPROVED** [review]
I would leave physical in the definitions. This looks good to me.
Discussed on the 10/6 UPheno call

**Review by rays22 — APPROVED** [review]

**Pull request merged**